x-pack/plugins/observability_solution/observability_onboarding/public/plugin.ts

```diff
@@ -120,6 +120,42 @@ export class ObservabilityOnboardingPlugin
         },
         visibleIn: [],
       });
+
```

Comment:
Do we still need the logic above to get the `observability_onboarding.experimental_onboarding_flow_enabled'` config? This now should be handled by the new URL path, right?

Reply:
We still need to for A/B testing capabilities.